integration-test/src/test/java/org/uniprot/store/indexer/search/uniprot/CCCofactorSearchIT.java

```diff
@@ -128,31 +127,6 @@ void findCofactorWithChebiName2() {
         assertThat(retrievedAccessions, contains(Q6GZX3, Q6GZY3));
     }
 
-    @Test
-    void findCofactorWithChebiNameEvidence() {
-        String query =
-                query(
-                        searchEngine
-                                .getSearchFieldConfig()
-                                .getSearchFieldItemByName("cc_cofactor_chebi"),
-                        "phosphate");
-        String evidence = "ECO_0000256";
-        query =
-                QueryBuilder.and(
-                        query,
-                        query(
-                                searchEngine
-                                        .getSearchFieldConfig()
-                                        .getSearchFieldItemByName("ccev_cofactor_chebi"),
-                                evidence));
```

Comment:
We could just remove ccev_cofactor_chebi from query instead of deleting the whole test. i have commented the same comment at other place, please take a look at other tests as well.

Reply:
The test above findCofactorWithChebiName2() already cover this use case with ccev_cofactor_chebi only. So, I removed the entire test to avoid duplication.